ledger/acctupdates.go

```diff
@@ -941,6 +941,13 @@ func (au *accountUpdates) totalsImpl(rnd basics.Round) (totals ledgercore.Accoun
 	return
 }
 
+// latestTotalsImpl returns the totals of all accounts for the most recent round, as well as the round number
+func (au *accountUpdates) latestTotalsImpl() (basics.Round, ledgercore.AccountTotals, error) {
+	offset := len(au.deltas)
+	rnd := au.dbRound + basics.Round(len(au.deltas))
+	return rnd, au.roundTotals[offset], nil
```

Comment:
Might be more straight-forward to do `au.roundTotals[len(au.roundTotals) - 1]`

Reply:
What you wrote is correct; but it's less consistent with the rest of the code that is using the `len(au.deltas)`.
There is also the theoretical risk of having a negative value that doesn't apply here.